apps.yaml

```diff
@@ -51,6 +51,27 @@ aiidalab-qe:
         version: '23.10'
         video_installation_url: https://drive.google.com/file/d/1TjzFB9SEkFn4E_TaJoBB3FMVJhhiy8BR/preview
         video_demonstration_url: https://drive.google.com/file/d/1-Cxx1agmyHtkfmk-Rb5-lQEGXh9cCDG8/preview
+aiidalab-qe-xas:
+    categories:
+        - technology-aiida
+        - technology-aiidalab
+        - quantum
+    git_url: https://github.com/aiidalab/aiidalab-qe.git
+    metadata:
+        authors: P. N. O. Gillespie, X. Wang, M. H. A. Bertràn, N. Marzari, E. Molinari, G. Pizzi, D. Prezzi
+        affiliations: CNR-NANO S3, Paul Scherrer Institute, Università di Modena e Reggio Emilia, Ecole Polytechnique Fédérale de Lausanne, Swiss Federal
+            Laboratories for Materials Science and Technology.
+        description: |
+            An extension to the AiiDALab-QE app for DFT calculations with Quantum
+            ESPRESSO for computing X-ray absorption near-edge spectra (XANES).
+        logo: https://raw.githubusercontent.com/aiidalab/aiidalab-qe/main/docs/source/_static/logos/QE.jpg
+        state: development
+        title: AiiDAlab-QE XAS App
+        external_url: https://github.com/aiidalab/aiidalab-qe
+        documentation_url: https://aiidalab-qe.readthedocs.io/howto/xas.html
+        version: 24.04.02a
+        video_installation_url: https://drive.google.com/file/d/1TjzFB9SEkFn4E_TaJoBB3FMVJhhiy8BR/preview
+        video_demonstration_url: https://drive.google.com/file/d/1m4imcAzaCJabL9Rjjal4GqqcVwGkIqmU/preview
```

Comment:
Can you use this link instead, so that the video is hosted on our end:
https://drive.google.com/file/d/1aoMfAabynAbkXkoktFBJrsKKxTtrFuHs/preview

Reply:
this is for video demonstration